src/main/java/sirius/kernel/nls/NLS.java

```diff
@@ -1189,25 +1189,26 @@ private static Double tryParseMachineFormat(String value) {
 
     @SuppressWarnings("unchecked")
     private static <V> V parseDatesFromUserString(Class<V> clazz, String value, String lang) {
+        boolean invertMonthAndDay = "fr".equals(NLS.getCurrentLang());
```

Comment:
Hm... not sure if this is worth it for this particular case. I would leave it as is for now, but I'm open for discussion

Reply:
Stupid questions:
1. Why `NLS.getCurrentLang()` here? What is the `lang` parameter of the method?
2. If the parameter was originally considered for British (as opposed to American?) dates, why do we 'only' have french here?

Reply:
1. true, that is unnecessary since the `lang` parameter already provides the current language --> fixed
2. We only use "en" as the language code for English and don't discern between en-gb or en-us (at least not that I am aware of). Apparently, there was some need for this parameter, as it was introduced here (https://github.com/scireum/sirius-kernel/pull/264). It seems, it just hasn't been used actively except for in the corresponding unit test.

Reply:
Thanks! 👍